### PR TITLE
30 min cache on Content Modules

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

File: apps/api/models/content-module/mapper.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { airtableToModel } from './domain'
 //  Mapper function called findByName that performs the transaction and calls the airtableToModel function on
 //  the return values.
 
-const cache = { ttl: 1000 * 60 }
+const cache = { ttl: 1000 * 60 * 30 }
```
Comment (craigspaeth) [inline apps/api/models/content-module/mapper.js]: Can we add `refreshAhead: true` here too so the cache stays warm?

Comment (lionsdwarf) [reply apps/api/models/content-module/mapper.js]: 👍

File: apps/api/models/content-module/mapper.js [hunk 2]
```diff
@@ -7,13 +7,14 @@ import { airtableToModel } from './domain'
...
 
 export const findByName = async name => {
   const records = await at.find({
     table: 'contentModules',
     filter: `name = '${name}'`,
-    cache
+    cache,
+    refreshAhead: true
```
Comment (craigspaeth) [inline apps/api/models/content-module/mapper.js]: This is an option [on cache](https://github.com/findOctave/cortex/blob/master/lib/caching-fetch/index.js#L22), so it goes inside the `cache =` part.

Comment (lionsdwarf) [reply apps/api/models/content-module/mapper.js]: DOH 🤦‍♂ 
Fixed :)

**Pull request merged**